data/processing/input_latency.html

```
@@ -155,20 +155,39 @@
     const associatedEvents = latency.associatedEvents;
 
     for (const event of associatedEvents) {
-      if (event.hasOwnProperty('parentContainer') && event.parentContainer.name === 'CrRendererMain')
+      if (event.title === 'RenderWidgetInputHandler::OnHandleInputEvent' &&
+	  event.args.hasOwnProperty('event') &&
+	  event.args.event === LATENCY_EVENT)	  
         return event.parentContainer.getProcess().pid;
     }
```

What happens if we don't return anything here, is this checked?

---

I guess this would be a DCHECK scenario. This function should always return something, if it doesn't something is very wrong! 

If it doesn't return anything then the mapper will crash for this specific trace and will output some error messages in the 'failures' portion of the results and then continue to the next trace. I can add a check for this and just have it skip the latency event if we can't get the id of the thread that handled the event. WDYT?

---

Is it possible to assert(condition) in a mapper? I think failing loudly w/ documentation would be great to do if possible. If not, yeah, it will fail somewhere down the line, but it's not clear where so IMO it would be nice to have documentation in the code (i.e. an assert).

---

Added an assert function that throws an error. That error will be logged in the output of the mapper.